spec/system/nav_bar_system_spec.rb

```diff
@@ -7,10 +7,9 @@
 
   it 'shows admin button as admin' do
     login_as_user(:admin)
-    visit('/')
 
     within(page.find('pzsh-menu')) do
-      expect(page).to have_text('Admin').twice
+      expect(page.body.downcase).to have_text('admin', count: 4)
```

Comment:
Why did this change from twice to count 4?

Reply:
Same question :grin: and why the downcase?